app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -387,9 +394,15 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
     override fun onPause() {
         daxDialog = null
         logoHidingListener.onPause()
+        dismissDownloadFragment()
```

Comment:
I made the rest of the code safe in terms of fragment recreation so we definitely don't need this. 
I went with removing it for usability reasons, I was testing my changes and found it awkward to leave the app and return to it later with the dialog still up. The dialog feels like a decision the user should make immediately and I thinks it's a better experience if the dialog is dismissed if the user leaves the screen. It's an edge case that probably won't impact many users so that's not a strong opinion on my side. If you feel really strongly about keeping it, I'm happy to go with that.

Reply:
I agree, it's an edge case and worst case scenario user only needs to click again on the link, like you said. I'm good keeping it like this.